src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -805,17 +805,27 @@ function addMcqOption(questionNumber) {
 }
 
 function removeMcqOption(index, questionNumber) {
-    idOfQuestion = '#form_editquestion-' + questionNumber;
-    idSuffix = (questionNumber > 0) ? ("-" + questionNumber) : "";
-    if(questionNumber == -1){
-        idSuffix = "--1";
-    }
-
-    $("#mcqOptionRow-"+index+idSuffix).remove();
-    
-    if($(idOfQuestion).attr('editStatus') == "hasResponses") {
-        $(idOfQuestion).attr('editStatus', "mustDeleteResponses");
-    }
+	idOfQuestion = '#form_editquestion-' + questionNumber;
+	idSuffix = (questionNumber > 0) ? ('-' + questionNumber) : '';
+
+	if (questionNumber == -1) {
+	    idSuffix = '--1';
+	}
+
+	var $thisRow = $('#mcqOptionRow-' + index + idSuffix);
+
+	// count number of child rows the table have and - 1 because of add option button
+	var numberOfOptions = $thisRow.parent().children('div').length - 1;
+
+	if (numberOfOptions <= 2) {
+	  alert('Too little choices for Multiple-choice (single answer) question. Minimum number of options is: 2.');
```

Comment:
Using alerts is probably a bad idea.
When the user tries to remove an option and there are <= 2 options, he is probably trying to clear the text and write something else. Maybe do that instead?